### PR TITLE
gl_detail_and_queries

### PR DESCRIPTION
**Pull request by CPAanalytics**

### CODE REVIEW

**Comment by CPAanalytics** [comment]
Added the gl_detail class.
Added ability to create advanced queries using get_by_query().
Updated Readme.

**Review by Shwetabhk — APPROVED** [review]

File: sageintacctsdk/apis/api_base.py [hunk 1]
```diff
@@ -361,3 +362,96 @@ def get_all(self, field: str = None, value: str = None, fields: list = None):
...
+                for operator, field, value in or_filter:
+                    formatted_filter['or'].setdefault(operator, {}).update({'field': field, 'value': value})
+            else:
+                formatted_filter = {}
+                for operator, field, value in or_filter:
+                    formatted_filter.setdefault(operator, {}).update({'field': field, 'value': value})
+
+        if formatted_filter:
+            data['query']['filter'] = formatted_filter
+        print(data)
```
Comment (ashwin1111) [inline sageintacctsdk/apis/api_base.py:434]: Please remove this print statement

File: sageintacctsdk/apis/gl_detail.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+"""
+Sage Intacct GL Detail
+"""
+from typing import Dict
+
+from .api_base import ApiBase
+
+
+class GLDetail(ApiBase):
+    """Class for AP Payments APIs."""
```
Comment (ashwin1111) [inline sageintacctsdk/apis/gl_detail.py:10]: """Class for GL Detail APIs."""

**Review by ashwin1111 — CHANGES_REQUESTED** [review]
@CPAanalytics couple of super minor comments, please fix it off and then we're good to merge and release.

**Comment by CPAanalytics** [comment]
Made these corrections.  Thanks for the opportunity to contribute.  I expect to use this SDK quite a bit so I may have some more pull requests.

**Review by Shwetabhk — APPROVED** [review]

**Comment by Shwetabhk** [comment]
https://github.com/fylein/sageintacct-sdk-py/releases/tag/1.6.0
@CPAanalytics this has been released in version `1.6.0`
https://pypi.org/project/sageintacctsdk/

**Pull request merged**